src/main/scala/com/kifi/franz/SimpleSQSQueue.scala

```diff
@@ -15,5 +16,10 @@ class SimpleSQSQueue(protected val sqs: AmazonSQSAsync, val queue: QueueName, pr
 
 class JsonSQSQueue(protected val sqs: AmazonSQSAsync, val queue: QueueName, protected val createIfNotExists: Boolean = false) extends SQSQueue[JsValue] {
   protected implicit def asString(json: JsValue) = Json.stringify(json)
-  protected implicit def fromString(s: String) = Json.parse(s)
+  protected implicit def fromString(s: String) = {
+    Try(Json.parse(s)) match {
```

Comment:
Completely independent of the merit of the change itself, creating a `Try` and then immediately unwrapping it seems a bit unnecessary to me. Cleaner:
```
try {
  Json.parse(s)
} catch {
   case ex => JsUndefined(ex.getMessage)
}
```